res/values/strings.xml

```diff
@@ -942,6 +962,8 @@
     <string name="no_spellcheck_suggestions_found">No spelling suggestions found.</string>
     <string name="show_window">Show Window</string>
     <string name="login_socks5_experimental_warning">SOCKS5 support is currently experimental. Please use at your own risk. If you type in an address in the e-mail field, there will be DNS lookup that won\'t get tunneled through SOCKS5.</string>
+    <!-- context: this key is used whenever there is a link to the about deltachat desktop dialog -->
+    <string name="about">About</string>
```

Comment:
in german, that would be translated to "Über" which is already ugly - and not even mistranslated as "Ungefähr" which may still easily happy - and we also had that already. we should not open that can of worms again, see more detailed discussions about that eg. just recently at https://github.com/deltachat/deltachat-desktop/issues/2904 - including better working alternatives.

i am a bit picky about that string as it is one that is pretty much visible and directly related to "Delta Chat". i'd like to avoid sth as "they cannot even create a title for their program information" ;)

Reply:
so rather use `global_menu_help_about_desktop`?

Reply:
<img width="185" alt="Bildschirmfoto 2022-11-16 um 18 44 23" src="https://user-images.githubusercontent.com/18725968/202254484-5c2a07dc-338c-4a1b-9ba3-684114a520a5.png">

Reply:
better. but i'd remove the first line completely now, that looks pretty redundant. if you really want to have a direct link, add it after "about delta chat". but i suggest to remove it completely here, that shortcut seems to be unneeded, what is the usecase? if even hidden that way? you get to the web easily from the about dialog.

Reply:
please make an issue for this on the desktop repo if you want this addressed